widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -368,7 +373,7 @@ internal class CallView(
             if (isActivated) activatedContentDescription else notActivatedContentDescription
 
         floatingActionButton.isActivated = isActivated
-        floatingActionButton.contentDescription = stringProvider.getRemoteString(contentDescription)
+        floatingActionButton.setLocaleContentDescription(contentDescription)
```

Comment:
I agree with you. Karl also suggested re-loading the whole activity instead of separate strings. So I have shortly looked into it but that also will have some problems.

I want to finish this solution for now and then try to improve it. One option is to improve on the current view/view-state/view-controller logic that is already there but not yet on the level where it is easy just to reload the locales.

Reply:
I think reloading the whole activity is a bad idea. We could replace texts but control/trigger them from a single place to minimize the chance of missing the upgrade step. 
Like I described it here - https://github.com/salemove/android-sdk-widgets/pull/1060#discussion_r1672369458. 
If you're interested in more details, we can have a short discussion on this topic.